packages/design-system/src/components/FilterTag/FilterTag.tsx

```diff
@@ -0,0 +1,104 @@
+import { ClearIcon } from '../ClearIcon';
+import React from 'react';
+import classNames from 'classnames';
+
+const actionableKeys = ['Enter', 'Backspace', 'Delete'];
+const idFromValue = (value: string) => `${value}-tag`;
+
+export interface FilterTagProps {
+  /**
+   * Id for filter tag button. If not provided, a unique id will be automatically generated and used.
+   */
+  id?: string;
+  /**
+   * Class to be applied to the outer `<div>` that contains filter tag button.
+   */
+  className?: string;
+  /**
+   * Text for the filter tag
+   */
+  label: string;
+  /**  
+   *
+   Value for filter tag 
+   */
+  value: string;
```

Comment:
I wonder how the `value` for filter tag is used because if a function is passed in `onClear` prop, why would the calling function want to get that value back? Wouldn't we just run that function on handle click?

Reply:
Because the React component using this probably has a state being managed of which items are currently filtered. We would call back their onDelete function with the value so they could then remove this item from its state.

Reply:
Ah, managing the state, thanks.

Reply:
@pwolfert Actually... is this really true above? Or could it all be handled on the parent component side without needing to be passed back the value?

Reply:
It's technically true that it can be handled without passing any value. I had made it this way in so that I didn't have to create a new handler function for each of these filter tag items. Most of the time I was iterating over an array when rendering them. If it seems too arbitrary to pass the value, we can work around it 🤷‍♂️ . It's not a deal breaker without it.

Reply:
We probably should keep the `value` prop as there's an existing use case to help track the filter tag items

Reply:
I think I'm going to remove value. It seems like an unnecessary extra layer.

Reply:
How would the applications track it then? Would returning the `id` on `onClick` be helpful?

Reply:
The applications themselves can just handle the value in onDelete without us needing to take it in and return it